source/pages/Home/Panel/components/TransactionDetails.tsx

```diff
@@ -188,32 +199,37 @@ export const TransactionDetails = ({ transactionType, transactionDetails }) => {
         <Disclosure>
           {({ open }) => (
             <>
-              <Disclosure.Button
-                className={`${
-                  open ? 'rounded-t-md' : 'rounded-md'
-                } mt-3 p-2 flex justify-between items-center  w-full border border-bkg-3 bg-bkg-1 cursor-pointer transition-all duration-300 text-xs`}
-              >
-                To
-                <Icon
-                  name="select-down"
+              <div className="px-6">
+                <Disclosure.Button
                   className={`${
-                    open ? 'transform rotate-180' : ''
-                  } mb-1 text-brand-white`}
-                />
-              </Disclosure.Button>
-
-              <Disclosure.Panel>
-                <div className="flex flex-col pb-2 px-2 w-full text-brand-white text-sm bg-bkg-3 border border-t-0 border-bkg-4 rounded-lg rounded-t-none transition-all duration-300">
-                  {Object.values(newRecipients).length &&
-                    renderAddresses(newRecipients)}
-                </div>
-              </Disclosure.Panel>
+                    open ? 'rounded-t-md' : 'rounded-md'
+                  } mt-3 py-2 px-2 flex justify-between items-center  w-full border border-bkg-3 bg-bkg-1 cursor-pointer transition-all duration-300 text-xs`}
+                >
+                  To
+                  <Icon
+                    name="select-down"
+                    className={`${
+                      open ? 'transform rotate-180' : ''
+                    } mb-1 text-brand-white`}
+                  />
+                </Disclosure.Button>
+              </div>
+
+              <div className="px-6">
+                <Disclosure.Panel>
+                  <div className="flex flex-col pb-2 px-2 w-full text-brand-white text-sm bg-bkg-3 border border-t-0 border-bkg-4 rounded-lg rounded-t-none transition-all duration-300">
+                    {Object.values(newRecipients).length &&
+                      renderAddresses(newRecipients)}
+                  </div>
+                </Disclosure.Panel>
+              </div>
             </>
           )}
         </Disclosure>
       )}
+      {copied && showSuccessAlert()}
```

Comment:
Can be turned into a `useEffect` with `copied` as a dependency. See `AccountHeader` copy.